.github/workflows/ci-artifacts.yml

```diff
@@ -14,10 +14,7 @@ jobs:
     runs-on: windows-latest
     steps:
       - name: clone git-sdk-64
-        shell: bash
-        run: git clone --bare --depth=1 --filter=blob:none --single-branch -b ${REF#refs/heads/} https://github.com/${{github.repository}}
-        env:
-          REF: ${{github.ref}}
+        run: git clone --bare --depth=1 --filter=blob:none --single-branch -b ${{github.ref_name}} https://github.com/${{github.repository}}
```

Comment:
It [does default to PowerShell](https://github.com/dscho/git-sdk-64/actions/runs/3481226868/jobs/5822040171#step:2:3), but since this is a simple call with no environment variable expansion, We could leave it at that?